src/app/researcher/study/request/[memberSlug]/study-proposal-form.tsx

```diff
@@ -45,80 +32,95 @@ export const StudyProposalForm: FC<{
-                            {agreementFileUpload}
-                            <FileInput
-                                name="agreementDocument"
-                                component={Anchor}
-                                aria-label="Upload Agreement Document"
-                                placeholder="Upload Document"
-                                clearable
-                                accept=".doc,.docx,.pdf"
-                                {...studyProposalForm.getInputProps('agreementDocument')}
-                            />
-                        </Group>
-                    </Stack>
-                </GridCol>
-            </Grid>
+                    </Group>
+
+                    <Group align="flex-start">
+                        <Text w={'20%'} fw="bold">
+                            Study Description
+                        </Text>
+                        <FileInput
+                            w={'30%'}
+                            name="descriptionDocument"
+                            leftSection={fileUpload}
+                            aria-label="Upload Study Description Document"
+                            placeholder="Upload Study Description Document"
+                            clearable
+                            accept=".doc,.docx,.pdf"
+                            {...studyProposalForm.getInputProps('descriptionDocument')}
+                        />
+                    </Group>
+
+                    <Group align="flex-start">
+                        <Text w={'20%'} fw="bold">
+                            IRB Document
+                        </Text>
+                        <FileInput
+                            w={'30%'}
+                            leftSection={irbFileUpload}
+                            {...studyProposalForm.getInputProps('irbDocument')}
+                            name="irbDocument"
+                            aria-label="Upload IRB Document"
+                            placeholder="Upload IRB Document"
+                            clearable
+                            accept=".doc,.docx,.pdf"
+                        />
+                    </Group>
+
+                    <Group align="flex-start">
+                        <Text w={'20%'} fw="bold">
+                            Agreement Document
+                        </Text>
+                        <FileInput
+                            w={'30%'}
+                            leftSection={agreementFileUpload}
+                            name="agreementDocument"
+                            aria-label="Upload Agreement Document"
+                            placeholder="Upload Agreement Document"
+                            clearable
+                            accept=".doc,.docx,.pdf"
+                            {...studyProposalForm.getInputProps('agreementDocument')}
+                        />
+                    </Group>
+                </Stack>
+            </Stack>
```

Comment:
Yepyep these are similar changes to the ones on #150.

Similarities:
- Broad reshaping of form into `<Stack><Group><Text/><Input/></Group><Group>...</Group></Stack>`

Differences:
- Text & Input width handling (I do fixed minwidth for Text and per-element width for Inputs, you do 20% and 30% for Text & Input respectively)
- You added the SVG inside the `FileInput` components which I think is a good move
- Specific fixes for our cards

Where are the Figma designs you're trying to imitate? I'm just vibe coding myself lmao

Reply:
yeah figma files [are here](https://www.figma.com/design/icLzqqjan9qZuK7T1JnZyk/AE-Beluga-simple-2025?node-id=3151-5236&t=VUCadDafa9OKD3rk-4) but semi-useless cause they're out of date and we've moved the code files section around

im aslo pushing for just using the native fileinput and not styling it as a link no response from devika though, so im planning on just keeping it as the file upload. we can always modify styles later, but this keeps everything lined up properly
![Screenshot 2025-04-18 at 12 22 03 PM](https://github.com/user-attachments/assets/d30f93e0-2cad-4bbb-9d21-13954f5491aa)

Reply:
im also not married to the width percentages, i dont want to have fixed widths because of responsiveness but maybe theres a better way... i originally had a grid layout which seemed to work okay, but 🤔

Reply:
I had the same problem with `FileInput` for accessibility; got permission from Iris to use the native component https://openstax.atlassian.net/browse/OTTER-149?focusedCommentId=16567